### DISCUSSION TITLE
Devlake project stuck on pending, nothing starts, logs empty

### DISCUSSION THREAD

**Discussion by pedraamn** [apache/devlake#6558] [Q&A] [answered]
Pretty much the title. For more context, I deleted all the other projects so this is the only one. Whenever I try to collect data basically nothing happens. Any ideas?

<img width="1728" alt="image" src="https://github.com/apache/incubator-devlake/assets/77109389/2e7194f1-7981-40ec-b496-1bdb040bf318">

**Comment by pedraamn** [comment]
Here is a relevant log from the pod: `127.0.0.1 - - [04/Dec/2023:20:55:31 +0000] "POST /api/blueprints/8/trigger HTTP/1.1" 400 290 "http://127.0.0.1:22344/projects/esg-dora?tab=status&tabId=blueprint" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36" "-"`

**Comment by klesh** [comment]
@pedraamn Hi, can you open up the Deveoper Toolbar and switch to the Network tab then share the response of the URL you mentioned? thanks.

**Comment by pedraamn** [comment, marked as answer]
This was solved. The issue was the old github PAT I was using had expired. I ended up trying a bunch of things, including completely reinstalling the helm deployment to the cluster. When I had to set up the connections again and tried the PAT it was showing expired so I made a new one. I wish there was an easier way to realize this was the issue. @klesh
